content/posts/ossprey-announcement/index.md

```diff
@@ -0,0 +1,75 @@
+---
+
+Open Source Software (OSS) powers everything from academic research to mission-critical infrastructure at tech giants. Yet, over **90% of OSS projects experience stagnation or abandonment**. The question is no longer whether OSS matters, but _how to ensure its sustainability_.
+
+## Meet OSSPREY
+
+The OSSPREY paper has been accepted in the [Automated Software Engineering 2025 Tool Demonstration Track](https://conf.researchr.org/track/ase-2025/ase-2025-tool-demonstration-track).
+
+![OSSPREY Icon](https://oss-prey.github.io/OSSPREY-Website/static/images/favicon.ico)
+
+**OSSPREY (Open Source Software PRoject sustainabilitY tracker)** is an intelligent, modular, and research-backed platform that enables open source maintainers to anticipate and address project health issues before they become critical. Developed by the DECAL Lab at UC Davis, OSSPREY uniquely combines real-time GitHub data scraping, socio-technical network modeling, machine learning forecasting, and a curated knowledge base of interventions grounded in software engineering research.
+
+By analyzing both technical contributions (e.g., commits, file changes) and social signals (e.g., issue discussions, collaboration patterns), OSSPREY delivers comprehensive month-by-month sustainability forecasts. What truly sets it apart is its ReACT recommendation engine, which maps empirical research findings to tailored, actionable strategies maintainers can deploy to counteract emerging risks—such as declining contributor activity, fragmented communication, or stagnant issue resolution.
+
+> “OSSPREY bridges the gap between academic sustainability metrics and real-world decision making by providing both forecasts and evidence-based recommendations.”
+
+## Why OSSPREY Matters
+
+Traditional metrics like issue count or commit frequency don’t tell the full story. OSSPREY moves beyond snapshots and offers:
+
+- **Real-time monitoring** of GitHub projects using socio-technical networks.
+- **Month-by-month sustainability forecasts** powered by a transformer model with 94% F1-score accuracy.
+- **Actionable ReACTs** — evidence-based interventions curated from 186 empirical studies.
+- **Interactive network visualizations** showing communication and technical dependencies.
+
+## How It Works
+
+OSSPREY’s pipeline includes four key modules:
+
+- **Scraper:** Rust-based module collects commits, issues, and metadata monthly via GitHub GraphQL API.
+- **Network Generator:** Builds social and technical networks to map collaboration and file ownership.
+- **Forecaster:** Transformer model forecasts future sustainability based on historical network features.
+- **ReACT-Recommender:** Suggests empirically validated strategies when sustainability dips.
+
+![OSSPREY dashboard with forecast trajectories, socio-technical networks, and actionable suggestions](https://raw.githubusercontent.com/OSS-PREY/OSSPREY-Website/refs/heads/main/static/images/ossprey-dashboard.png)
```

Comment:
The text in this image is pretty tiny. Could you please try increasing the font size by maybe 30% and re-taking the screenshot?